Basic-Car-Maintenance/Shared/Settings/Views/AddVehicleView.swift

```diff
@@ -14,8 +14,12 @@ struct AddVehicleView: View {
     @State private var name = ""
     @State private var make = ""
     @State private var model = ""
+    @State private var year = ""
+    @State private var color = ""
+    @State private var VIN = ""
+    @State private var licensePlateNumber = ""
     private var isVehicleValid: Bool {
-        !name.isEmpty && !make.isEmpty && !model.isEmpty
+        !name.isEmpty && !make.isEmpty && !model.isEmpty && !year.isEmpty && !color.isEmpty && !VIN.isEmpty && !licensePlateNumber.isEmpty
```

Comment:
Can you actually remove these? I think it's fine if these aren't included, but the `name`, `make`, and `model`, are the only ones that are required

This also makes the properties optional

Reply:
Sure

Reply:
Done @mikaelacaron